packages/registry/package.json

```diff
@@ -25,7 +25,7 @@
     "chai": "4.3.6",
     "dotenv": "16.0.0",
     "ethers": "5.5.4",
-    "hardhat": "2.8.4",
+    "hardhat": "2.9.1",
```

Comment:
I had to normalize all the `hardhat` versions because this caused an error when writing the tests of `contracts` which didn't load the plugin types using lerna.